src/main/java/com/gitblit/wicket/pages/MyTicketsPage.java

```diff
@@ -278,6 +320,12 @@ public void populateItem(final Item<TicketSort> item) {
 			}
 		}
 
+        if (!qb.containsField(Lucene.repository.name()) && !StringUtils.isEmpty(repository)) {
```

Comment:
Resetting back to *all* doesn't work right.

If you add something like this it will:
```java
        if (noneChoice != currentRepository && !qb.containsField(Lucene.repository.name()) ...
```